README.md

```diff
@@ -15,3 +15,13 @@ $ cp config.toml.sample config.toml
 $ pip install -e ".[dev]"
 $ REDASH_API_KEY=OCZccH...FhB4cT python main.py config.toml
 ```
```

Comment:
Here we could remove the Redash key and have a paragraph about env variables and the `.env` file

Reply:
Yep! I'll add that.